.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ jobs:
     docker:
       - image: oasislabs/rust:latest
     environment:
-        PATH: /root/.cargo/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+        PATH: /root/.cargo/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/$HOME/.local/bin
```

Comment:
does https://github.com/oasislabs/toolstate/blob/master/.circleci/config.yml#L7 displease you for some reason?